src/definitions/modules/progress.js

```diff
@@ -423,11 +423,19 @@ $.fn.progress = function(parameters) {
             else {
               var firstNonZeroIndex = -1;
               var lastNonZeroIndex = -1;
+              var percentSum = 0;
```

Comment:
while we are at it, this can be cleaned up a bit more, because you already have implemented a helper for the sum functionality
```javascript
var percentSum = module.helper.sum(values);
```

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI/pull/625/commits/730f0c12011d18b14ae760bcd443ec46571b1f09